sources/packages/api/src/services/disbursement-schedule-service/disbursement-schedule-service.ts

```diff
@@ -252,4 +254,55 @@ export class DisbursementScheduleService extends RecordDataModelService<Disburse
       .where("application.id = :applicationId", { applicationId })
       .execute();
   }
+
+  /**
+   * Get the list of disbursement schedules for a given location as COE.
+   * @param locationId
+   * @returns List of COE for given location.
+   */
+  async getCOEByLocation(
+    locationId: number,
+    upcomingCOE?: boolean,
+  ): Promise<DisbursementSchedule[]> {
+    const coeThresholdDate = addDays(new Date(), 21);
```

Comment:
Please use the constant already defined under: api\src\utilities\system-configurations-constants.ts

Reply:
Constant value used.